plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/render/FileSystemRenderer.java

```diff
@@ -28,7 +28,6 @@
 import org.eclipse.jdt.annotation.NonNull;
```

Comment:
I am worried that this might loose the view for other entry paths through the RendererManager. If every thing checks out this is a good simplification. I'm not sure that the regression test suite is sufficient to catch it though. Perhaps we should still set the view if it has not already been set in the renderer options.

Reply:
The view is set through another UI element and its value is stored in the initial creation of renderer options elsewhere. This method's purpose is to ensure all artifacts are from the same branch. The loaded branchToken is not expected to have a view inside, hence the already set view in the renderer options is being set to sentinel (-1) every time.